R/plotNorm.R

```diff
@@ -38,20 +38,38 @@ plotNorm <- function(DGEdata,
                      plotType = "canvasXpress",
```

Comment:
plotNorm - Is still accepting a counts matrix or DGEobj. This needs to be accept only DGEobj.

Reply:
done